lib/pure/typetraits.nim

```diff
@@ -15,6 +15,25 @@
 import std/private/since
 export system.`$` # for backward compatibility
 
+type SomeSparseEnum* = concept x ## Type class for sparse enum's.
```

Comment:
Would `enum and not Ordinal` work?

Reply:
no, but `(not Ordinal) and enum` does work, so I've now changed it to that; thanks